tests/integration/cwd-priv.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+load helpers
+
+# This test must be particular with how it's run. The user that runs it must have the privileges
+# to chown a directory it creates away from itself, but not have CAP_DAC_OVERRIDE to override
+# the fact it was chowned away. However, it must also be able to clean up.
+# Thus, this test must be skipped if the UID running the test is root, or the user doesn't have sudo privileges.
```

Comment:
I don't quite get it. Can this test be run as root, and then drop the privs to a non-root user when it needs to?

Asking because the way it is right now, our CI never runs it.

Reply:
unfortunately, I wasn't able to figure out how to run the test as a non privileged user